packages/opencensus-core/src/metrics/utils.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LabelValue} from './export/types';
+
+/**
+ * Returns a string(comma separated) from the list of label values.
+ *
+ * @param  {LabelValue[]} labelValues The list of the label values.
+ * @returns {string} The hashed label values string.
+ */
+export function hashLabelValues(labelValues: LabelValue[]): string {
+  if (labelValues.length === 0) {
+    return '';
+  }
+  if (labelValues.length > 1) {
+    labelValues = labelValues.sort((obj1, obj2) => {
+      if (obj1.value > obj2.value) {
```

Comment:
you can simplify it by using `+-` operators instead of this `ifs`

Reply:
SGTM, done.

Reply:
still the same

Reply:
I changed to ```+(obj1.value > obj2.value) || -(obj1.value < obj2.value) || 0;```  https://github.com/census-instrumentation/opencensus-node/pull/207/files#diff-4ecd32f36720fa9f740ab01cc93ee360R31. Let me know if something else is expected.

Reply:
```suggestion
      export function hashLabelValues(labelValues: LabelValue[]): string {
        return labelValues.map(lv => lv.value).sort().join(this.COMMA_SEPARATOR)
      }
```
You can use + operator if you need desc ordered

Reply:
done, thanks

Reply:
Please remove this redundant
```suggestion
     if (labelValues.length === 0) {
       return '';
    }
```

Reply:
Done.